opm/input/eclipse/Schedule/UDQ/UDQASTNode.hpp

```diff
@@ -135,6 +142,49 @@ class UDQASTNode
                                   const UDQContext& context) const;
 
     void func_tokens(std::set<UDQTokenType>& tokens) const;
+
+    /// Publish collection of requisite objects needed to evaluate this node.
```

Comment:
populate?

Reply:
> populate?

Sure.  I've pushed an update to that effect.